innotter/services/permissions/permissions.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework import permissions
+from innotter.models import User
+
+
+# Allows only administrators to execute methods; others - only read requests.
+class IsAdminOrReadOnly(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+        return bool(request.user and request.user.is_staff)
```

Comment:
You also need to check for user role as we added an enum in our model for it and we do not use default django is_staff
```suggestion
        return request.user and request.user.role == User.Roles.ADMIN
```